src/handlers/relnotes.rs

```diff
@@ -64,13 +63,15 @@ pub async fn handle(ctx: &Context, event: &Event) -> anyhow::Result<()> {
     }
 
     if let IssuesAction::Labeled { label } = &e.action {
-        if ["relnotes", "relnotes-perf", "finished-final-comment-period"]
-            .contains(&label.name.as_str())
+        if label.name == "relnotes"
+            || label.name == "relnotes-perf"
+            || label.name == "finished-final-comment-period"
+                && e.issue
+                    .labels
+                    .iter()
+                    .any(|label| label.name == "disposition-merge")
```

Comment:
Could you add parentheses or perhaps extract `let is_fcp_merge = ...`?

Reply:
Applied.